src/recorder.js

```diff
@@ -121,15 +122,22 @@ Recorder.prototype.initSourceNode = function( sourceNode ){
   });
 };
 
+Recorder.prototype.loadWorker = function() {
+  if ( this.encoder ) {
```

Comment:
I think this be more concise as `if ( !this.encoder ) { ... }`

Reply:
I'll update